Remotion/Web/Development.WebTesting/AspNetErrorPageDetection.cs

```diff
@@ -0,0 +1,68 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+// 
+// The re-motion Core Framework is free software; you can redistribute it 
+// and/or modify it under the terms of the GNU Lesser General Public License 
+// as published by the Free Software Foundation; either version 2.1 of the 
+// License, or (at your option) any later version.
+// 
+// re-motion is distributed in the hope that it will be useful, 
+// but WITHOUT ANY WARRANTY; without even the implied warranty of 
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the 
+// GNU Lesser General Public License for more details.
+// 
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+// 
+using System.Runtime.Serialization;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.Extensions;
+using Remotion.Web.Development.WebTesting.Utilities;
+
+namespace Remotion.Web.Development.WebTesting
+{
+  /// <summary>
+  /// Provides methods for detecting an ASP.NET error page.
+  /// </summary>
+  public static class AspNetErrorPageDetection
```

Comment:
Since the detection cannot be used with async-postbacks on Smartpage at this time, we should document that this api is only intended for synchronous postbacks or GET-requests at this time. And we should add a ticket to support async-postbacks to be resolved at a later date.

Reply:
I tested errors and asynchronous postbacks, and it just did nothing -> there was no update. Is there even a visible error page that can be detected on a failed asynchronous postback?

Reply:
Yes, if you use the SmartPage or WxePage as a baseclass. The plain page create a console error (or swallows the error by default)

You can repro this by opening the test page for BocAutoComplete, and in the first BocAutoComplete (has an auto-postback) select a value without an icon (those generate exceptions). Or you build a testpage off SmartPage :)